src/composite-disposable.coffee

```diff
@@ -51,8 +51,12 @@ class CompositeDisposable
   #
   # * `disposable` {Disposable} instance or any object with a `.dispose()`
   #   method.
-  add: (disposable) ->
-    @disposables.push(disposable) unless @disposed
+  add: (disposables...) ->
+    unless @disposed
+      if disposables.length is 1 and Array.isArray(disposables[0])
```

Comment:
I'm a little unclear why this `Array.isArray` check is needed here, can you elaborate a bit?

Why can't you always just iterate over `disposables`?